.cirun.yml

```diff
@@ -0,0 +1,8 @@
+runners:
+  - name: long-runner
+    cloud: aws
+    instance_type: t3.nano
```

Comment:
This instance is way too small for doing anything useful, it might just hung up during job, when you would do heavier things like docker. I would suggest start with a larger machine and maybe later try to see if you can get your job done in a smaller one. The aws machine closest to GitHub Actions machine would be `t2.large`.

Reply:
```suggestion
    instance_type: t2.large
```